### PR TITLE
Display the URL cache size

### PR DESCRIPTION
**Pull request by waliid**
# Description

Self-explanatory.

clear_cache

# Changes made

- Added `UrlCacheView`.
- Updated `SettingView`.

# Checklist

- [x] APIs have been properly documented (if relevant).
- [x] The documentation has been updated (if relevant).
- [x] New unit tests have been written (if relevant).
- [x] The demo has been updated (if relevant).
- [x] The playground has been updated (if relevant).

### CODE REVIEW

File: Demo/Sources/Settings/SettingsView.swift [hunk 1]
```diff
@@ -8,6 +8,35 @@ import AVFoundation
...
+                .buttonStyle(.borderless)
+#endif
+            Spacer()
+            Text(urlCacheSize)
+                .font(.footnote)
+                .foregroundColor(.red)
+        }
+        .onAppear {
+            updateCacheSize()
+        }
```
Comment (defagos) [inline Demo/Sources/Settings/SettingsView.swift:27]:
```suggestion
        .onAppear(perform: updateCacheSize)
```

Comment (defagos) [inline Demo/Sources/Settings/SettingsView.swift:27]: The `currentDiskUsage` property seems to be KVObservable. You can therefore likely use `.onReceive(URLCache.shared.publisher(for: \.currentDiskUsage))` to not only get the initial value, but also potential updates which could happen while the view is visible on screen.

Comment (waliid) [reply Demo/Sources/Settings/SettingsView.swift:27]: Thank you for your suggestion. It was the first thing I attempted. Unfortunately, it didn't work. I'll give it another try and inform you of the results.

Comment (defagos) [reply Demo/Sources/Settings/SettingsView.swift:27]: You're right. The current implementation is good enough, then, thanks.

**Review by defagos — APPROVED** [review]

**Pull request merged**